signer/src/stacks/api.rs

```diff
@@ -552,18 +584,22 @@ impl StacksInteract for StacksClient {
             })
             .await?;
         // As of this writing the RPC response includes exactly 3 estimates
-        // (the low, medium, and high priority estimates). We want to use
-        // the high priority estimate (the max) to make sure that the
-        // transaction gets confirmed as quickly as possible.
-        let fee_estimate = fee_estimates
-            .estimations
-            .iter()
-            .map(|estimate| estimate.fee)
-            // TODO(366): have priority be configurable.
-            .max()
-            .unwrap_or(DEFAULT_TX_FEE);
+        // (the low, medium, and high priority estimates). It's note worthy
+        // if this changes so we log it but the code here is robust to such
+        // a change.
+        let num_estimates = resp.estimations.len();
+        if num_estimates != 3 {
```

Comment:
Does it EVER return an estimate that is not 3? D:

Reply:
We'll find out when it's deployed!